package.json

```diff
@@ -27,10 +27,10 @@
     "version": "npm publish"
   },
   "devDependencies": {
-    "@thisismanta/semantic-version": "^8.0.0",
-    "@vitest/coverage-v8": "^1.2.2",
-    "lefthook": "^1.6.1",
-    "typescript": "^5.3.3",
-    "vitest": "^1.2.2"
+    "@thisismanta/semantic-version": "^9.1.0",
```

Comment:
would recommend configuring https://github.com/amannn/action-semantic-pull-request so that maintenance is out of our hands

Reply:
I'm using this for my other open source projects so I don't mind maintaining.

This library does more than just validating commit messages though. It provides another command to run `npm version` and `git push` on CI. This is to avoid boilerplates.

I don't mind replacing it for Taskworld, if you have a strong opinion.